### PR TITLE
proposal: new project application process

### PR DESCRIPTION
**Pull request by MylesBorins**
First pass at documenting proposal

### CODE REVIEW

**Comment by MylesBorins** [comment]
Updated to merge with the existing new project checklist proposal PTAL

**Comment by jorydotcom** [comment]
LGTM!! TY @MylesBorins !

**Comment by MylesBorins** [comment]
https://github.com/openjs-foundation/bootstrap/pull/165 implements the changes to `PROJECT_PROGRESSION.md`

File: proposals/stage-1/PROJECT_ONBOARDING/NEW_PROJECT_APPLICATION.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
+# Project Name
+
+## Project Champion
+
+Who is the main point of contact during the application process?
+
+## CPC Champion (optional)
+
+If you have a champion from the CPC please identify them. If you do not currently have one we will identify someone to support your project.
```
Comment (keywordnew) [inline proposals/stage-1/PROJECT_ONBOARDING/NEW_PROJECT_APPLICATION.md]: Any thoughts about this person being selected from voting members, or also non-voting members?

File: proposals/stage-1/PROJECT_ONBOARDING/NEW_PROJECT_APPLICATION.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+Links to social media accounts
+
+## Existing Financial Sponsorship
+
+Does your project currently receive funds? Who do they come from and what are the funds used for?
+
+## Infrastructure Needs or Requests
+
+What needs will your project have from the foundation?
```
Comment (keywordnew) [inline proposals/stage-1/PROJECT_ONBOARDING/NEW_PROJECT_APPLICATION.md]: Could be helpful here to have a list of possible needs the Foundation can fulfill. Extra points if we know how asking for an item would affect the New Project Application's chances or pace of progression :)

File: proposals/stage-1/PROJECT_ONBOARDING/NEW_PROJECT_APPLICATION.md (hunk 1 above)
Comment (MylesBorins) [reply proposals/stage-1/PROJECT_ONBOARDING/NEW_PROJECT_APPLICATION.md]: My idea was it would be whomever volunteered

Comment (keywordnew) [reply proposals/stage-1/PROJECT_ONBOARDING/NEW_PROJECT_APPLICATION.md]: Comment resolved during #168

File: proposals/stage-1/PROJECT_ONBOARDING/NEW_PROJECT_APPLICATION.md (hunk 2 above)
Comment (keywordnew) [reply proposals/stage-1/PROJECT_ONBOARDING/NEW_PROJECT_APPLICATION.md]: Comment resolved during #168

File: proposals/stage-1/PROJECT_ONBOARDING/README.md [hunk 3]
```diff
@@ -3,30 +3,51 @@
...
-We need to share a workflow and understanding of what steps are required to bring in projects to the OpenJS Foundation. This checklist is an attempt to organize our ideas and our work to make that happen effectively.
+We need to share a workflow and understanding of what steps are required to bring in projects to the OpenJS Foundation. This process is an attempt to organize our ideas and our work to make that happen effectively. This process should be explicit and consistent to make applications fair for all projects.
 
-## What is neccessary to complete this proposal
+## Unresolved Question
 
-Agreement on the initial checklist items; interim owners/helpers for each of the tasks.
+* Should there be a quiet period?
+* Should all emeritus projects be expected to go through this process?
+* Should projects be able to enter as `Impact Level`?
```
[the remark this answers is not in the document]
Comment (mhdawson) [reply proposals/stage-1/PROJECT_ONBOARDING/README.md]: I agree that as long as the CPC believes it is a appropriate then entering at the 'impact' level should be ok.

File: proposals/stage-1/PROJECT_ONBOARDING/README.md [hunk 4]
```diff
@@ -3,30 +3,51 @@
...
+Agreement on the initial checklist items; interim owners/helpers for each of the tasks.
 
+- [x] [Template for Project Applications](./NEW_PROJECT_APPLICATION.md)
+- [x] [Onboarding Check List](./NEW_PROJECT_APPLICATION.md#onboarding-checklist)
+- [ ] [Update to Project Progression](https://github.com/openjs-foundation/bootstrap/pull/158)
+- [ ] Potential changes to CPC Charter
+- [ ] Discussion/approval by bootstrap committee
+- [ ] Discussion/approval by the Board
+- [ ] Adoption by the CPC
+- [ ] Run existing projects through the process
```
[the remark this answers is not in the document]
Comment (MylesBorins) [reply proposals/stage-1/PROJECT_ONBOARDING/README.md]: This is to dogfood process and make sure that it is both reasonable and that current projects are accomplishing the requirements

File: proposals/stage-1/PROJECT_ONBOARDING/NEW_PROJECT_APPLICATION.md [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+- [ ] Update legal copyright notice on project website and github
+- [ ] Add OpenJS Foundation logo to project website
+- [ ] Add Project logo to OpenJS Foundation website; update PROJECTS.md file
+- [ ] Transfer logomark to the OpenJS Foundation
+- [ ] If project is using crowdfunding platforms, add disclaimer to platforms
+- [ ] Identify individuals from the project to join the CPC
+- [ ] Document project and foundation contacts for:
+  * marketing & social media
+  * infrastructure
+  * legal/governance help
```
[the remark this answers is not in the document]
Comment (MylesBorins) [reply proposals/stage-1/PROJECT_ONBOARDING/NEW_PROJECT_APPLICATION.md]: captured in readme

**Pull request merged**